### PR TITLE
Register

### PR DESCRIPTION
**Pull request by codermuskanj**
Register Page

### CODE REVIEW

**Review by harshgoel05 — CHANGES_REQUESTED** [review]
Please remove the `.eslintcache` file and add it in
.gitignore

File: src/App.js [hunk 1]
```diff
@@ -2,12 +2,14 @@ import logo from './logo.svg';
...
 import Header from './Components/Header';
 import LogIn from './Components/Login';
+import Register from './Components/Register';
 
 function App() {
   return (
     <div className="App">
-      <LogIn/>
+      <Register/>
     </div>
```
Comment (harshgoel05) [inline src/App.js:11]: Don't just directly add the component over here. Use react routing

File: src/Components/Register.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
+import React, { Component } from "react";
+import { Button, Grid, InputAdornment, TextField } from "@material-ui/core";
+import styles from './Headercss.module.css';
+
+export class Register extends Component {
```
Comment (harshgoel05) [inline src/Components/Register.js:5]: Use functional components instead of class components

File: src/Components/Register.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+            sm={4}
+            alignItems="center"
+            direction="column"
+            justify="space-between"
+            style={{ padding: 10 }}
+          >
+            <div />
+
+            <div
+              style={{
```
Comment (harshgoel05) [inline src/Components/Register.js:39]: Create a style sheet and don't use inline styling

**Review by harshgoel05 — CHANGES_REQUESTED** [review]

**Comment by codermuskanj** [comment]
Screenshot from 2020-12-07 00-44-26
Screenshot from 2020-12-07 00-45-16

**Pull request merged**